api/nakadi-event-bus-api.yaml

```diff
@@ -1152,3 +1550,108 @@ definitions:
           should have a description.
     required:
       - publishing_status
+
+  PaginationLinks:
+    description: contains links to previous and next pages of items
+    type: object
+    properties:
+      prev:
+        $ref: '#/definitions/PaginationLink'
+      next:
+        $ref: '#/definitions/PaginationLink'
+
+  PaginationLink:
+    description: URI identifying another page of items
+    type: object
+    properties:
+      href:
+        type: string
+        format: uri
+        example: '/subscriptions?offset=20&limit=10'
+
+parameters:
+  EventTypeName:
+    name: name
+    in: path
+    description: EventType name to get events about
+    type: string
+    required: true
+
+  BatchLimit:
+    name: batch_limit
+    in: query
+    description: |
+      Maximum number of `Event`s in each chunk (and therefore per partition) of the stream.
+
+      * If 0 or unspecified will buffer Events indefinitely and flush on reaching of
+      `batch_flush_timeout`.
+    type: integer
+    format: int32
+    required: false
+    default: 1
+
+  StreamLimit:
```

Comment:
Why is this parameter needed? It seems like it's a way to emulate a polling request; if so maybe it's better to have a regular GET with count API.

Also what happens, a) when the limit is reached, b) the limit can't be reached because there isn't enough events to reach it?